tests/tests/regression_fuzzer_tests/acceleration_assert.json

```diff
@@ -0,0 +1 @@
+{"error_type": "SCHEDULE", "code": 500, "error": "{\"type\":\"core:assert_error\",\"cause\":\"INTERNAL\",\"stackTrace\":[\"EnvelopePhysics.java:14\",\"EnvelopePhysics.java:54\",\"ConstrainedEnvelopePartBuilder.java:102\",\"EnvelopeAcceleration.java:25\",\"AbstractAllowanceWithRanges.java:397\",\"AbstractAllowanceWithRanges.java:315\",\"AbstractAllowanceWithRanges.java:280\",\"AbstractAllowanceWithRanges.java:257\",\"AbstractAllowanceWithRanges.java:187\",\"AbstractAllowanceWithRanges.java:129\",\"StandaloneSim.java:95\",\"StandaloneSim.java:58\",\"StandaloneSimulationEndpoint.java:85\",\"FkRegex.java:157\",\"FkRegex.java:228\",\"FkChain.java:72\",\"TkFork.java:98\",\"TkFallback.java:89\",\"TkFallback.java:57\",\"TkFallback.java:69\",\"TkWrap.java:58\",\"TkSlf4j.java:110\",\"BkBasic.java:127\",\"BkBasic.java:102\",\"BkSafe.java:49\",\"BkWrap.java:51\",\"BkParallel.java:87\",\"ThreadPoolExecutor.java:1136\",\"ThreadPoolExecutor.java:635\",\"Thread.java:833\"],\"trace\":[]}", "infra_name": "small_infra", "path_payload": {"infra": 94, "name": "foo", "steps": [{"duration": 0, "waypoints": [{"track_section": "TA1", "offset": 1856.6077154118593}]}, {"duration": 1, "waypoints": [{"track_section": "TA4", "offset": 25.049733930557206}]}]}, "schedule_payload": {"timetable": 74, "path": 16446, "schedules": [{"train_name": "foo", "labels": [], "departure_time": 0, "allowances": [{"allowance_type": "engineering", "begin_position": 86.71793651828453, "end_position": 92.01725738112799, "value": {"value_type": "percentage", "percentage": 19.78715063935094}, "capacity_speed_limit": 0.12096469825170542, "distribution": "LINEAR"}], "initial_speed": 0, "rolling_stock": "253"}]}}
```

Comment:
Is it really usefull to keep the error code and the stack trace. We could add a field `description` and explain what this test is doing.

Reply:
The point is to be able to just cp the error files generated by the fuzzer, in this context the error is relevant. We can remove it manually but I don't think there's any reason to, it can be useful to know what bug this test reproduced. 

We could add a description, but there's not a lot to describe, those are just generated regression tests. "This test reproduces the assertion bug" doesn't add much information.